packages/flutter/test/widgets/text_selection_test.dart

```diff
@@ -652,6 +653,54 @@ void main() {
     expect(editableText.selectionOverlay!.toolbarIsVisible, isFalse);
   });
 
+  // TODO(justinmc): Selection.
+  testWidgets('Mouse drag selects and cannot drag cursor', (WidgetTester tester) async {
+    // Regression test for https://github.com/flutter/flutter/issues/102928
+    final TextEditingController controller = TextEditingController(
+      text: 'I love flutter!',
+    );
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Scaffold(
+          body: EditableText(
+            controller: controller,
+            focusNode: FocusNode(),
+            backgroundCursorColor: Colors.white,
+            cursorColor: Colors.white,
+            style: const TextStyle(),
+            //selectionControls: CustomTextSelectionControls(),
```

Comment:
Is the command there for a purpose?

Reply:
Nope I forgot it there, thanks for catching that.